src/main/java/fr/cnes/sonar/plugins/icode/check/ICodeSensor.java

```diff
@@ -26,6 +27,7 @@
 import fr.cnes.sonar.plugins.icode.model.XmlHandler;
 import fr.cnes.sonar.plugins.icode.rules.ICodeRulesDefinition;
 import fr.cnes.sonar.plugins.icode.settings.ICodePluginProperties;
+import org.apache.commons.lang.SystemUtils;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.apache.commons.lang.SystemUtils'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/lequal/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fr.cnes.sonarqube.plugins%3Asonaricode&pullRequest=31&issues=AWXTgLl8ashydt7kJTGl&open=AWXTgLl8ashydt7kJTGl)

[ISSUE_KEY]: AWXTgLl8ashydt7kJTGl